pom.xml

```diff
@@ -71,6 +71,12 @@
             <version>3.2.13-37.vf3411c9828b9</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.github.docker-java</groupId>
+            <artifactId>docker-java-transport-httpclient5</artifactId>
+            <version>3.2.13</version>
```

Comment:
I agree. Updates of docker-java-api-plugin are requested in [jenkinsci/docker-java-api-plugin/pull/26](https://github.com/jenkinsci/docker-java-api-plugin/pull/26). This will be removed when the PR will be merged.

Reply:
Hi @mat1e,

PR jenkinsci/docker-java-api-plugin/pull/26 has been merged on Oct 12, 2022, so will you update this PR to remove this dependency ?